### PR TITLE
`reference_path` and `image_path` now expect absolute path in configs

### PR DESCRIPTION
**Pull request by vladsavelyev**
We switched to generating config [by merging templates in AR](https://github.com/populationgenomics/analysis-runner/pull/559), with templates generated by Pulumi in [cpg-infra](https://github.com/populationgenomics/cpg-infrastructure/pull/20) as well as by the CI in corresponding [images](https://github.com/populationgenomics/images/pull/56/files) and [references](https://github.com/populationgenomics/references/pull/2) repositories. When running outside of our infra, absolute paths are expected now, so modifying AIP configs accordingly.

### CODE REVIEW

File: reanalysis/interpretation_runner.py [hunk 1]
```diff
@@ -332,7 +332,7 @@ def main(
         # generate the jobs which run VEP & collect the results
         vep_jobs = add_vep_jobs(
             b=get_batch(),
-            input_siteonly_vcf_path=to_path(input_path),
+            vcf_path=to_path(input_path),
```
Comment (MattWellie) [inline reanalysis/interpretation_runner.py:335]: https://github.com/populationgenomics/production-pipelines/blob/main/cpg_workflows/jobs/vep.py#L30 ?
```suggestion
            input_siteonly_vcf_path=to_path(input_path),
```

File: reanalysis/reanalysis_global.toml [hunk 2]
```diff
@@ -41,23 +40,19 @@ default_timeout = 6000
...
-cpg_workflows = 'cpg_workflows:latest'
-hail = 'cpg_workflows:latest'
-picard = 'picard_samtools:v0'
-picard_samtools = 'picard_samtools:v0'
+vep = 'australia-southeast1-docker.pkg.dev/cpg-common/images/vep:105.0'
+gatk = 'australia-southeast1-docker.pkg.dev/cpg-common/images/gatk:4.2.6.1'
+cpg_workflows = 'australia-southeast1-docker.pkg.dev/cpg-common/images/cpg_workflows:latest'
+picard = 'australia-southeast1-docker.pkg.dev/cpg-common/images/picard:2.27.4'
+samtools = 'australia-southeast1-docker.pkg.dev/cpg-common/images/samtools:1.16.1'
+hail = 'australia-southeast1-docker.pkg.dev/cpg-common/images/cpg_workflows:latest'
```
Comment (MattWellie) [inline reanalysis/reanalysis_global.toml:48]:
```suggestion
vep = 'australia-southeast1-docker.pkg.dev/cpg-common/images/vep:105.0'
cpg_workflows = 'australia-southeast1-docker.pkg.dev/cpg-common/images/cpg_workflows:latest'
hail = 'australia-southeast1-docker.pkg.dev/cpg-common/images/cpg_workflows:latest'
```

**Review by MattWellie — COMMENTED** [review]
@vladsavelyev can you check the add_vep_jobs call change

File: reanalysis/reanalysis_global.toml (hunk 2 above)
Comment (MattWellie) [reply reanalysis/reanalysis_global.toml:48]: Trimming this as AIP doesn't use many images - they were just copied in bulk from the template

File: reanalysis/interpretation_runner.py (hunk 1 above)
Comment (vladsavelyev) [reply reanalysis/interpretation_runner.py:335]: Sorry 🤦

**Review by MattWellie — APPROVED** [review]

**Pull request merged**